pkg/local/lib/parser.go

```diff
@@ -377,34 +378,49 @@ func ParseLV(line string) (*LV, error) {
 }
 
 // ParseVG parse volume group
-func ParseVG(line string) (*VG, error) {
-	// vgs --units=b --separator="<:SEP:>" --nosuffix --noheadings -o vg_name,vg_size,vg_free,vg_uuid,vg_tags,pv_count --nameprefixes -a
-	fields, err := parse(line, 6)
+func ParseVGs(data []byte) ([]VG, error) {
+	// vg_name,vg_size,vg_free,vg_uuid,vg_tags,pv_count
+	vgsOut := struct {
+		Report []struct {
+			Vg []struct {
+				VgName  string `json:"vg_name"`
+				VgSize  string `json:"vg_size"`
+				VgFree  string `json:"vg_free"`
+				VgUUID  string `json:"vg_uuid"`
+				VgTags  string `json:"vg_tags"`
+				PvCount string `json:"pv_count"`
+			} `json:"vg"`
+		} `json:"report"`
+	}{}
+	err := json.Unmarshal(data, &vgsOut)
```

Comment:
make sure it works properly without any VGs

Reply:
tested and it works fine